File: Find_the_Median.js
function findMedian(arr) {
  var arrayLength = arr.length;

  if (arrayLength % 2 == 0) return false;

  arr.sort(function(a, b) {
    return a - b;
  });
  return arr[Math.floor(arrayLength / 2)];
}

console.log(findMedian([9, 7, 1, 2, 3, 4, 5]));

File: find_the_median.rb
# frozen_string_literal: true

def findMedian(arr)
  return false if arr.size.even?

  arr.sort!

  p arr[(arr.size/2).floor]
end

findMedian([9, 7, 1, 2, 3, 4, 5]) # expected result is 4 
